app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::Base
+ def home
```

Comment:
```suggestion
 def home; end
```
tương tự cho các method khác nhé em